modules/support_dashboard/src/Service/TriageExtractService.php

```
@@ -0,0 +1,259 @@
+    }
+
+    private function rejectUnauthenticated(string $clientIp): TriageExtractResult
+    {
+        // Same table and same blind-index shape as the statistics intake's
+        // per-address limit, under its own purpose so the two never
+        // count each other's attempts.
+        $ipHash = $this->encryption->blindIndex('support_triage_ip:' . $clientIp);
+        $since = (new \DateTimeImmutable('-' . self::UNAUTHENTICATED_WINDOW_MINUTES . ' minutes'))
+            ->format('Y-m-d H:i:s');
+        if ($this->rateLimits->countSince($ipHash, $since) >= self::UNAUTHENTICATED_JOURNAL_LIMIT) {
+            return TriageExtractResult::rejected(TriageExtractResult::REJECT_UNAUTHENTICATED);
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'Modules\SupportDashboard\Service\TriageExtractResult' in method 'rejectUnauthenticated'. (undefined)

(StaticAccess)